exercises/exercise3.js

```diff
@@ -1,12 +1,444 @@
 // Set this assignment to true if you do want to use it.
-module.exports.ACTIVATE_BOT = false;
+module.exports.ACTIVATE_BOT = true;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
+    label: "Introduction",
+    prompt: "Start",
     handler: function () {
-      return 'Good Evening to You!';
+      return 'Hi there! Before we get started, I just need to point out that I don\'t like one word answers. \n So instead of answering with "Bob", try "It\'s Bob". Why don\'t you start by telling me your name?';
```

Comment:
Fantastic return Kevin - do you have a UX background? Your return on this made the next prompt intuitive! Way to work around a problem!

Reply:
Got a design degree but been in UX/UI and static sites for about three years now.